### DISCUSSION TITLE
Connect an authentication provider

### DISCUSSION THREAD

**Discussion by mitya-shepelev** [lobehub/lobehub#1667] [Development Question]
Hi implement this authentication provider to make it easier for users to log in  https://www.loginradius.com/

**Comment by arvinxx** [comment]
We have support Auth0、 Azure AD, if you want to integration with this provider, welcome to PR

**Comment by mitya-shepelev** [comment]
These providers are no longer supported in Russia!

**Reply by arvinxx** [reply]
I'm sorry to hear it
